test/lvm/lvm_test.go

```diff
@@ -0,0 +1,119 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT
+//go:build !windows
+
+package acceptance
+
+import (
+	"log"
+	"testing"
+	"time"
+
+	"github.com/aws/amazon-cloudwatch-agent-test/environment"
+	"github.com/aws/amazon-cloudwatch-agent-test/internal/common"
+	"github.com/aws/amazon-cloudwatch-agent-test/test/metric"
+	"github.com/aws/amazon-cloudwatch-agent-test/test/metric/dimension"
+	"github.com/aws/amazon-cloudwatch-agent-test/test/status"
+	"github.com/aws/amazon-cloudwatch-agent-test/test/test_runner"
+)
+
+const (
+	namespace    = "LVMTest"
+	agentRuntime = 3 * time.Minute
+)
+
+var envMetaDataStrings = &(environment.MetaDataStrings{})
+
+func init() {
+	environment.RegisterEnvironmentMetaDataFlags(envMetaDataStrings)
+}
+
+type LVMTestRunner struct {
```

Comment:
Why are these in the style of the test suite but we only have one test per suite. I feel like the test should stand on its own or we combine multiple into a single suite.

Reply:
It's not in the test suite style but using `base_test_runner` which is used by a suite. This is to have the same test runner structure for better readability.